src/main/java/org/apache/rocketmq/dashboard/service/impl/TopicServiceImpl.java

```diff
@@ -61,16 +61,28 @@
     private RMQConfigure configure;
 
     @Override
-    public TopicList fetchAllTopicList(boolean skipSysProcess) {
+    public TopicList fetchAllTopicList(boolean skipSysProcess, boolean skipRetryAndDlq) {
         try {
             TopicList allTopics = mqAdminExt.fetchAllTopicList();
             if (skipSysProcess) {
                 return allTopics;
             }
 
-            TopicList sysTopics = getSystemTopicList();
             Set<String> topics = new HashSet<>();
+            if (skipRetryAndDlq) {
```

Comment:
1. What will happen if both skipSysProcess and skipRetryAndDlq are true.
2. maybe we can handle skipSysProcess and skipRetryAndDlq together, for example use Java lambda stream.
```java
allTopics.getTopicList().stream().map((item)->{
                ...
            }).filter(new Predicate<String>() {
                @Override
                public boolean apply(@Nullable String input) {
                    ...
                }
            }).collect(Collectors.toSet());
```

Reply:
Yes,  need to take into account the fact that both skipSysProcess and skipRetryAndDlq are true，I will optimize this method using stream.